src/main/java/net/imagej/ops/features/hog/HistogramOfOrientedGradients2D.java

```diff
@@ -191,7 +191,7 @@ public void compute(RandomAccessibleInterval<T> in, RandomAccessibleInterval<T>
 		final CursorBasedChunk chunkable = new CursorBasedChunk() {
 
 			@Override
-			public void execute(int startIndex, int stepSize, int numSteps) {
+			public void execute(long startIndex, int stepSize, long numSteps) {
```

Comment:
Is there any value to `long stepSize`? Or do you think `int stepSize` is sufficient always?

Reply:
If changing it is easy, we might as well, since we are breaking the API anyway right now. Better to break once now, then need to break again in future, no?

If, in the process of changing it, you find the change becoming more involved than anticipated, then please abandon the effort and we can stick with `int`.

Reply:
Changing the interface was straight forward.  I just made the changes and rebased the commits.  Before merging I need to run some final due dillegence tests with > 4G images.  I'll get to that later today or first thing Monday.  In the mean time let me know if there is anything else I should look at.